hv_1/task_1.py

```diff
@@ -0,0 +1,25 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) 
+
+# 1 способ:
+n = int(input("Введите трехзначное число:"))
+summa = 0
+while n > 0:
+    x = n % 10
+    summa = summa + x
+    n = n // 10
```

Comment:
Это тоже можно записать короче.
```suggestion
    n //= 10
```